examples/v3io-kv.ipynb

```diff
@@ -0,0 +1,674 @@
+       "            <td>201</td>\n",
+       "            <td>2003-12-17 00:00:00.000</td>\n",
+       "            <td>8.9</td>\n",
+       "            <td>The Lord of the Rings: The Return of the King</td>\n",
+       "        </tr>\n",
+       "        <tr>\n",
+       "            <td>195</td>\n",
+       "            <td>1993-02-04 00:00:00.000</td>\n",
+       "            <td>8.9</td>\n",
+       "            <td>Schindler&#x27;s List</td>\n",
+       "        </tr>\n",
+       "        <tr>\n",
+       "            <td>152</td>\n",
+       "            <td>2008-07-18 00:00:00.000</td>\n",
+       "            <td>9.0</td>\n",
+       "            <td>The Dark Knight</td>\n",
+       "        </tr>\n",
+       "        <tr>\n",
+       "            <td>142</td>\n",
+       "            <td>1994-10-14 00:00:00.000</td>\n",
+       "            <td>9.2</td>\n",
+       "            <td>The Shawshank Redemption</td>\n",
+       "        </tr>\n",
+       "        <tr>\n",
+       "            <td>178</td>\n",
+       "            <td>1967-12-29 00:00:00.000</td>\n",
+       "            <td>8.8</td>\n",
+       "            <td>The Good, the Bad and the Ugly</td>\n",
+       "        </tr>\n",
+       "        <tr>\n",
+       "            <td>154</td>\n",
+       "            <td>1994-10-14 00:00:00.000</td>\n",
+       "            <td>8.8</td>\n",
+       "            <td>Pulp Fiction</td>\n",
+       "        </tr>\n",
+       "        <tr>\n",
+       "            <td>96</td>\n",
+       "            <td>1957-04-10 00:00:00.000</td>\n",
+       "            <td>8.9</td>\n",
+       "            <td>12 Angry Men</td>\n",
+       "        </tr>\n",
+       "        <tr>\n",
+       "            <td>202</td>\n",
+       "            <td>1974-12-18 00:00:00.000</td>\n",
+       "            <td>9.0</td>\n",
+       "            <td>The Godfather: Part II</td>\n",
+       "        </tr>\n",
+       "    </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "[(175, '1972-03-24 00:00:00.000', 9.1, 'The Godfather'),\n",
+       " (178, '2001-12-19 00:00:00.000', 8.8, 'The Lord of the Rings: The Fellowship of the Ring'),\n",
+       " (201, '2003-12-17 00:00:00.000', 8.9, 'The Lord of the Rings: The Return of the King'),\n",
+       " (195, '1993-02-04 00:00:00.000', 8.9, \"Schindler's List\"),\n",
+       " (152, '2008-07-18 00:00:00.000', 9.0, 'The Dark Knight'),\n",
+       " (142, '1994-10-14 00:00:00.000', 9.2, 'The Shawshank Redemption'),\n",
+       " (178, '1967-12-29 00:00:00.000', 8.8, 'The Good, the Bad and the Ugly'),\n",
+       " (154, '1994-10-14 00:00:00.000', 8.8, 'Pulp Fiction'),\n",
+       " (96, '1957-04-10 00:00:00.000', 8.9, '12 Angry Men'),\n",
+       " (202, '1974-12-18 00:00:00.000', 9.0, 'The Godfather: Part II')]"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%sql select * from $query_table"
```

Comment:
This is just to show that the data is readable with the schema that was created. Not sure if you have a better way, because this should be either spark or trino.
We could skip that "proof", but it does make the example more complete.